services/datamanager/datasync/sync.go

```diff
@@ -81,23 +82,43 @@ func NewManager(identity string, client v1.DataSyncServiceClient, logger logging
 	cancelCtx, cancelFunc := context.WithCancel(context.Background())
 	logger.Debugf("Making new syncer with %d max threads", maxSyncThreads)
 	ret := syncer{
-		partID:             identity,
-		client:             client,
-		logger:             logger,
-		cancelCtx:          cancelCtx,
-		cancelFunc:         cancelFunc,
-		arbitraryFileTags:  []string{},
-		inProgress:         make(map[string]bool),
-		syncErrs:           make(chan error, 10),
-		syncRoutineTracker: make(chan struct{}, maxSyncThreads),
-		captureDir:         captureDir,
+		partID:            identity,
+		client:            client,
+		logger:            logger,
+		cancelCtx:         cancelCtx,
+		cancelFunc:        cancelFunc,
+		arbitraryFileTags: []string{},
+		inProgress:        make(map[string]bool),
+		syncErrs:          make(chan error, 10),
```

Comment:
(off-diff comment about `syncer.Close`)

I see `syncer.Close` closes syncErrs. I'd expect it to close these other channels along with the new `filesToSync`. I don't think forgetting to close this has any material implications. So if there's any uncertainty, feel free to ignore for the sake of expediency.

Reply:
Added close for the chanel.